src/goPath.ts

```diff
@@ -95,4 +95,15 @@ function fileExists(filePath: string): boolean {
 
 export function clearCacheForTools() {
 	binPathCache = {};
+}
+
+/**
+ * Exapnds ~ to homedir in non-Windows platform and replaces ${workspaceRoot} token with given workspaceroot
+ */
+export function resolvePath(inputPath: string, workspaceRoot?: string): string {
+	if (!inputPath || !inputPath.trim()) return inputPath;
+	if (workspaceRoot) {
+		inputPath = inputPath.replace(/\${workspaceRoot}/g, workspaceRoot);
+	}
+	return (process.platform === 'win32' || !inputPath.startsWith('~')) ? inputPath : path.join(os.homedir(), inputPath.substr(1));
```

Comment:
Because ~ being the home dir is a POSIX thing. Doesn't apply to Windows.

But I see your point, if there was a workspace setting for go path using ~ checked in to source control, then the same workspace should work on Windows machine.

Thoughts?